Makefile

```diff
@@ -298,10 +298,10 @@ CUDA_LIB_DIR := $(patsubst %/,%,$(dir $(firstword $(CUDA_LIB_DIR))))
 CUDA_LIB_DIR_STUBS := $(CUDA_LIB_DIR)/stubs
 CUDA_BACKENDS = /gpu/cuda/ref /gpu/cuda/reg /gpu/cuda/shared
 ifneq ($(CUDA_LIB_DIR),)
-  $(libceed) : CPPFLAGS += -I$(CUDA_DIR)/include
-  $(libceed) : LDFLAGS += -L$(CUDA_LIB_DIR) -Wl,-rpath,$(abspath $(CUDA_LIB_DIR))
-  $(libceed) : LDFLAGS += -L$(CUDA_LIB_DIR_STUBS) -Wl,-rpath,$(abspath $(CUDA_LIB_DIR_STUBS))
-  $(libceed) : LDLIBS += -lcudart -lnvrtc -lcuda
+  $(libceeds) : CFLAGS += -I$(CUDA_DIR)/include
```

Comment:
CPPFLAGS was correct here. I have a tentative merge that I'm trying to test again (needed to switch machines). I'll push the merge tonight.